bindings/pyroot_legacy/JupyROOT/helpers/cppcompleter.py

```diff
@@ -61,6 +61,7 @@ class CppCompleter(object):
     TH1F
     TH1I
     TH1K
+    TH1L
```

Comment:
Could you please add this in the new JupyROOT too? It's in `bindings/JupyROOT`. Thank you!

Reply:
But JupyROOT does not have this example in the documentation. There is no TH1I being defined in JupyROOT?

the CppCompleter class in JupyROOT mentions the autocomplete example to TTreeF instead of TH1.

Reply:
Indeed, this changed in the new JupyROOT, so no need to do anything!